services/DevHome.Services.Core/Services/MicrosoftStoreService.cs

```diff
@@ -2,24 +2,25 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Management.Automation;
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
-using Serilog;
+using DevHome.Services.Core.Contracts;
+using Microsoft.Extensions.Logging;

```

Comment:
Why the change from Serilog?

Reply:
`Serilog` is just one logging service, if in the future we decided to change our logger or adding other ones, depending on Microsoft `ILogger` will help us achieve this without making changes in all files. This change should also be applied to all files in Dev Home, but I only did it for the new projects to avoid touching more files.

Related links:
- https://github.com/microsoft/devhome/blob/16eef45af935d65ca457b259531493124ec6bd94/src/App.xaml.cs#L103

Reply:
CC: @dkbennett

Reply:
This is goodness; it's still using Serilog. When the initial logging switch happened, I didn't want to convert everything over to Microsoft.Extenstions.Logging as that would have been loads more work and churn, so instead enabled that to happen later by adding Serilog to the services host in the line Amir mentions above. With that injected, anyone can now use Microsoft.Extensions.Logging on a per-service basis as needed. That's exactly what Amir is doing here, so this is good and expected. The logger is still writing to Serilog because it is one of the configured loggers in the service host.